borg/archiver.py

```diff
@@ -56,6 +56,9 @@ def print_verbose(self, msg, *args, **kw):
         msg = args and msg % args or msg
         logger.info(msg)
 
+    def print_status(self, status, path):
+        logger.debug("%1s %s", status, remove_surrogates(path))
```

Comment:
can we just agree on not misusing logging levels ever?

this is NOT debug output.